Pre release 0.4.0 fails tests

**Describe the bug**
Tests on the pre-release 0.4.0 tag fail.

```
~/Projects/aur/slides-git/src/slides main
❯ go test ./...
?   	github.com/maaslalani/slides	[no test files]
?   	github.com/maaslalani/slides/cmd	[no test files]
ok  	github.com/maaslalani/slides/internal/code	(cached)
ok  	github.com/maaslalani/slides/internal/file	(cached)
ok  	github.com/maaslalani/slides/internal/meta	(cached)
?   	github.com/maaslalani/slides/internal/model	[no test files]
--- FAIL: TestExecute (0.00s)
    --- FAIL: TestExecute/Replace (0.00s)
        execute_test.go:34: Invalid execution, want Replace, got 
FAIL
FAIL	github.com/maaslalani/slides/internal/process	0.001s
ok  	github.com/maaslalani/slides/styles	(cached)
FAIL
```

**To Reproduce**
Steps to reproduce the behavior:

```
git clone https://github.com/maaslalani/slides.git
cd slides
git checkout main
git pull
go test ./...
```

Alternatively, on Arch Linux:

```
git clone https://aur.archlinux.org/slides-git.git
cd slides-git
makepkg
```

Yields:

```
~/Projects/aur/slides-git master*
❯ makepkg
==> Making package: slides-git 0.4.0.r0.g742251d-1 (Sun 27 Jun 2021 09:40:40 PM EDT)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating slides git repo...
Fetching origin
From https://github.com/maaslalani/slides
 - [deleted]         (none)     -> node
 - [deleted]         (none)     -> preprocess
 - [deleted]         (none)     -> preprocessing
==> Validating source files with sha256sums...
    slides ... Skipped
==> Extracting sources...
  -> Creating working copy of slides git repo...
Switched to and reset branch 'makepkg'
==> Starting pkgver()...
==> Removing existing $pkgdir/ directory...
==> Starting build()...
==> Starting check()...
?   	github.com/maaslalani/slides	[no test files]
?   	github.com/maaslalani/slides/cmd	[no test files]
ok  	github.com/maaslalani/slides/internal/code	(cached)
ok  	github.com/maaslalani/slides/internal/file	0.002s
ok  	github.com/maaslalani/slides/internal/meta	(cached)
?   	github.com/maaslalani/slides/internal/model	[no test files]
--- FAIL: TestExecute (0.00s)
    --- FAIL: TestExecute/Replace (0.00s)
        execute_test.go:34: Invalid execution, want Replace, got 
FAIL
FAIL	github.com/maaslalani/slides/internal/process	0.001s
ok  	github.com/maaslalani/slides/styles	(cached)
FAIL
==> ERROR: A failure occurred in check().
    Aborting...
```

**Expected behavior**
Tests pass.

**Desktop (please complete the following information):**
 - OS: Arch Linux `5.12.12-arch1-1`
 - Golang: `go version go1.16.5 linux/amd64`

--- a/internal/process/execute_test.go
+++ b/internal/process/execute_test.go
@@ -16,7 +16,7 @@ func TestExecute(t *testing.T) {
 		},
 		{
 			block: Block{
-				Command: "sd Find Replace",
+				Command: "sed -e s/Find/Replace/g",
 				Input:   "Find",
 			},
 			want: "Replace",

